setup.cfg

```diff
@@ -143,6 +143,17 @@ testing =
     # of the control flow:
     pypytools
 
+    # mypy is a static type checker
+    mypy==0.910
+
+    # type stubs for mypy
+    types-backports
+    types-setuptools
+    types-six
+    types-requests
+    types-pyOpenSSL
```

Comment:
Type checking is not the same as running tests and has different dependencies. This is why it needs to live in a different requirements file. Also, I've started moving away from having the extras that are useless in runtime, env pins should live in dedicated constraints files.

Besides, you didn't add this to the CI and it should be present for this PR to make sense to be merged. I don't think that it's worth setting it up separately but we have pre-commit integrated so it'd be fine if you just move this to `.pre-commit-config.yaml`. This way, you won't need to add any other integration manually.

Reply:
any suggestions how to call that file?

Reply:
Probably `requirements/mypy.in` or `requirements/types.in`. Just make sure to use `.in` as I'm planning to integrate pip-tools later and Dependabot only supports pip-tools setups with pairs of `.in` and `.txt` files with the same base name.

Reply:
I never worked with pip-tools so can you please just tell me what to do here?

Reply:
Oh, you don't need to integrate it right now, it's out of the scope of the current PR. I'm planning to do that, I wouldn't request you to integrate something that is not yet streamlined in the repo. I was just trying to explain that you need to create a file called `requirements/mypy.in` following `requirements.txt` syntax.

Reply:
OTOH, if we're going to integrate this via pre-commit per https://github.com/cherrypy/cheroot/pull/385#discussion_r759814196, this file is not needed — the dependencies must be listed in `.pre-commit-config.yaml` as `additional_dependencies` under the respective mypy hooks.

Reply:
okay, as soon as the other discussion is done, I'll do that

Reply:
I know there was some python2 backport. `types`? Need to search for it but it's totally possible.